src/main/kotlin/Calculator.kt

```diff
@@ -1,3 +1,82 @@
-class Calculator(val A: Float, val B: Float) {
-    fun Plus(): Float{return A + B}
-}
\ No newline at end of file
+class Calculator {
+    fun plus(leftOperand: Double, rightOperand:Double) =  leftOperand + rightOperand
+    fun minus(leftOperand: Double, rightOperand: Double) = leftOperand - rightOperand
+    fun multiply(leftOperand: Double, rightOperand: Double) = leftOperand * rightOperand

```

Comment:
It must be not here, it's other tasks. It is the reason why that block u. U should finish them at first. And after u can finish this task